Chore/Chore.js

```diff
@@ -105,27 +105,55 @@ function runChoreIntroduction() {
         sendVirtualAssistantMessage('Okay, I will remove this from the list then...')
     }
 
+	if (isVar("ToyGirlfriend") && getVar("ToyGirlfriend"))
```

Comment:
Why aren't you using the global VARIABLE_SUB_HAS_GIRLFRIEND variable?

Reply:
the code pre-dates your renaming of all the variables...

Reply:
also this code works.  you don't populate this variable anywhere in your tree (via either variable name)